src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -1140,6 +1146,20 @@ function bp_core_admin_debug_information_add_help_tab() {
 					if ( document.readyState === "complete" ) {
 						bpHelpSidebarLinks = document.querySelector( \'.bp-help-sidebar-links\' ).closest( \'p\')
 						bpHelpSidebarLinks.style.display = \'none\';
+
+						document.querySelectorAll( \'#health-check-accordion-block-buddypress tr > td\' ).forEach( function( td, i ) {
```

Comment:
That's a nice hack! I initially looked for a way to do this without using js and could not find it.